content/guides/translation-connection/step3-interact-with-julia.md

```diff
@@ -0,0 +1,241 @@
+:::
+
+### Create a Translation attribute group
+
+First, in order to not mix these new product attributes with other Julia's product attributes, it is necessary that she creates **a specific attribute group** named `"Translations"`.
+
+To create this attribute group:
+1. Go to `Settings > Attribute groups`
+2. Click on `CREATE` at top right
+3. Add a new attribute group with the code: `Translations`.
+
+
+### Create Translation attributes
+
+Then Julia must create 6 new attributes in this attribute group:
+
+* `Translation submission name`
+* `Translation submitter`
+* `Translation queued`
+* `Translation locales`
+* `Translation due date`
+* `Translation status`
+
+:::tips
+To create an attribute:
+1. Go to `Settings > Attributes`
+2. Click on `CREATE ATTRIBUTE` at top right
+3. Choose the correct attribute type
+4. Then set the below information
+:::
+
+
+`Translation submission name`  
+**Attribute type:** Text  
+**Attribute code:** translationName  
+**Attribute group:** Translations  
+**Usable in grid:** Enabled  
+
+This attribute allows Julia to set a name to her translation project.  
+It also allows her to easily find this project with this name in the interface of your online Translation solution.
+
+`Translation submitter`  
+**Attribute type:** Simple select  
+**Attribute code:** translationSubmitter  
+**Attribute group:** Translations  
+**Usable in grid:** Enabled  
+
+This simple select attribute should contains in its `options` all possible contributor names. This attribute allows Julia to associate her name to her translation project.
+
+`Translation queued`  
+**Attribute type:** Yes/No (Boolean)  
+**Attribute code:** translationQueued  
+**Attribute group:** Translations  
+**Usable in grid:** Enabled  
+
+This attribute allows Julia to set if a product need to be translated.
+
+`Translation locales`  
+**Attribute type:** Multi select  
+**Attribute code:** translationLocales  
+**Attribute group:** translationLocales  
+**Usable in grid:** Enabled  
+
+This "Multi select" attribute should contains in its `options` all destination locales Julia can use to make her translation requests.  These same `locales` must of course also already exist in Akeneo PIM and in your online Translation tool.  
+This attribute allows Julia to choose one or more target locales for her translation project.
+
+`Translation due date`  
+**Attribute type:** Date  
+**Attribute code:** translationDate  
+**Attribute group:** Translations  
+**Usable in grid:** Enabled  
+
+This attribute allows Julia to indicate to translators when she would like her products to be translated.
+
+`Translation status`  
+**Attribute type:** Simple select  
+**Attribute code:** translationStatus  
+**Attribute group:** Translations  
+**Value per locale:** Enabled (in order to have a status for each locale)    
+**Read only:** Enabled (Can only be modified via API)   
+**Usable in grid:** Enabled    
+
```

Comment:
```suggestion
```
To delete

Reply:
I prefer your previous Idea: deleted this one.